backend/app/api/v1/integrations.py

```
@@ -9,101 +9,156 @@
     """Create a new organization integration."""
+    logger.info(f"Creating integration for user {user_id}, platform: {request.platform}")
+    
     try:
-        return await integration_service.create_integration(user_id, request)
+        integration = await integration_service.create_integration(user_id, request)
+        logger.info(f"Successfully created integration {integration.id} for user {user_id}")
+        return integration
+        
     except ValueError as e:
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
+        logger.warning(f"Validation error creating integration: {str(e)}")
```

Ruff (0.14.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)